public/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sky Events</title>
+  </head>
+
+  <body>
+    <h1>Sky Events</h1>
+    <hr />
+    <p id="message"></p>
+    <hr />
+    <div id="logon-register" style="display: none">
+      <button type="button" id="logon">login</button>
+      <button type="button" id="register">register</button>
+    </div>
+    <div id="logon-div" style="display: none">
+      <form>
+        <div>
+          <label for="email">email:</label>
+          <input type="email" id="email" />
+        </div>
+        <div>
+          <label for="password">password:</label>
+          <input type="password" id="password" />
+        </div>
+        <button type="button" id="logon-button">login</button>
+        <button type="button" id="logon-cancel">cancel</button>
+      </form>
+    </div>
+    <div id="register-div" style="display: none">
+      <form>
+        <div>
+          <label for="login-name">name:</label>
+          <input type="text" id="login-name" />
+        </div>
+        <div>
+          <label for="email1">email:</label>
+          <input type="email" id="email1" />
+        </div>
+        <div>
+          <label for="password1">password:</label>
+          <input type="password" id="password1" />
+        </div>
+        <div>
+          <label for="password2">verify password:</label>
+          <input type="password" id="password2" />
+        </div>
+        <button type="button" id="register-button">register</button>
+        <button type="button" id="register-cancel">cancel</button>
+      </form>
+    </div>
+    <div id="data" style="display: none">
+      <p id="data-message"></p>
+      <table id="data-table">
+        <tr id="data-table-header">
+          <th>Event</th>
+          <th>Name</th>
+          <th>
+            Date
+            <br />
+            (yyyy-mm-dd)
+          </th>
+          <th>Description</th>
+          <th colspan="2"></th>
+        </tr>
+      </table>
+      <button type="button" id="add-data">add event</button>
+      <button type="button" id="logoff">log off</button>
+    </div>
+    <div id="edit-data" style="display: none">
+      <form>
+        <div>
+          <label for="event">Event:</label>
+          <select id="event">
+            <option value="comet">comet</option>
+            <option value="asteroid">asteroid</option>
+            <option value="meteor_shower">meteor shower</option>
+            <option value="solar_eclipse">Solar Eclipse</option>
```

Comment:
These field values differ from the accepted enum defined in Data model in the backend. If I choose this options the backend is returning an error.